src-tauri/src/github/mod.rs

```diff
@@ -1,2 +1,179 @@
 pub mod pull_requests;
 pub mod release_notes;
+
+use app::constants::{APP_USER_AGENT, SECTION_ORDER};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+use ts_rs::TS;
+
+#[derive(Serialize, Deserialize, Debug, Clone, TS)]
+#[ts(export)]
+pub struct Tag {
+    name: String,
+}
+
+/// Wrapper type needed for frontend
+#[derive(Serialize, Deserialize, Debug, Clone, TS)]
+#[ts(export)]
+pub struct TagWrapper {
+    label: String,
+    value: Tag,
+}
+
+#[derive(Debug, Deserialize)]
+struct CommitInfo {
+    sha: String,
+    commit: Commit,
+}
+
+#[derive(Debug, Deserialize)]
+struct Commit {
+    message: String,
+}
+
+#[derive(Debug, Deserialize)]
+struct Comparison {
+    commits: Vec<CommitInfo>,
+}
+
+/// Get a list of tags on the FlightCore repo
+#[tauri::command]
+pub fn get_list_of_tags() -> Result<Vec<TagWrapper>, String> {
+    // Set the repository name.
+    let repo = "R2NorthstarTools/FlightCore";
```

Comment:
Shouldn't this be in constants package?

Reply:
Good catch, done in ee016b85d2d4b53c0ef1f0822bbd6b0e47f30ac7